runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridConfig.java

```diff
@@ -35,37 +36,52 @@
 	@ConfigItem(
 		keyName = "showItem",
 		name = "Show item",
-		description = "Show a preview of the item in the new slot"
+		description = "Show a preview of the item in the new slot",
+		position = 0
 	)
 	default boolean showItem()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showGrid",
-		name = "Show grid",
-		description = "Show a grid on the inventory while dragging"
+		keyName = "showHighlight",
+		name = "Highlight background",
+		description = "Show a green background highlight on the new slot",
+		position = 1
 	)
-	default boolean showGrid()
+	default boolean showHighlight()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showHighlight",
-		name = "Highlight background",
-		description = "Show a green background highlight on the new slot"
+			keyName = "showInventoryGrid",
```

Comment:
De-indent by one, there should only be one tab beyond the tab level of the `@ConfigItem`.